_sass/_py1.scss

```diff
@@ -0,0 +1,156 @@
+// debug
+.py1-wrapper * {
+    // border: solid 1px red;
+}
+
+.py1-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin-top: 10px;
+    p, span, a {
+	// everything seems set in Poppins?
+	font-family: $font-display;
+    }
```

Comment:
Good catch! ACM actually recently updated [its styleguide](https://design.uclaacm.com/) (after the mockup was made), so the correct font would be Open Sans. That's set in `$font-body`, but it should inherit from `body`!